lib/organisms/gravity-form/gravity-form.directive.js

```diff
@@ -2,8 +2,18 @@ angular
   .module('lnPatterns')
   .directive('lnOGravityForm', lnOGravityForm);
 
+/**
+ * TODO:
+ *    - Expand the list of supported input types (see comments in form parser)
+ *
+ *    - NOTE: The validation feedback / update mechanism isn't very 'Angular'
+ *      at the moment
+ *
+ *    - Migrate to use of ng-model for form data gathering
+ */
+
 /*@ngInject*/
```

Comment:
we will use ngAnnotate in our apps, however i think it makes things more reusable by others if we explicitly include the $inject's